portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -13,22 +13,31 @@
 // limitations under the License.
 
 package com.google.sps.data;
-
```

Comment:
nit: Leaving a space between your package name and your imports is common practice.

Reply:
Got it